### PR TITLE
LPS-120673 Graph configuration options more visible

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**
After doing some user tests in the Content Dashboard, we realize that the configuration was a little bit hidden and far from the Audit Graph, so with this pull request, we add a new button in the Audit Graph sheet, to make the configuration more visible. This configuration is the same (and works the same) as the option inside the kebab menu on top.

_Note_: To see these changes go to `Application Menu > Content Dashboard`.

**Screenshots**

Screenshot 2020-09-28 at 11 46 40
Screenshot 2020-09-28 at 11 46 51

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
Hey @liferay-frontend can you review this changes?
I just added a new button `<clay:button>` with `onClick` event to open the portlet configuration. Thanks 😄

**Review by julien — APPROVED** [review]

**Comment by julien** [comment]
For me the code change is OK, I deployed and checked and it's all working well.
Just leaving a comment here in case someone else wants to test and forward this.

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -31,7 +31,26 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
...
+						</clay:content-col>
+
+						<clay:content-col>
+							<span class="lfr-portal-tooltip" title="<%= LanguageUtil.get(request, "configure-chart") %>">
+								<clay:button
+									borderless="<%= true %>"
+									cssClass="component-action"
+									displayType="secondary"
+									icon="cog"
+									onClick="<%= portletDisplay.getURLConfigurationJS() %>"
```
Comment (jbalsas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp:48]: I always forget about this, which is terrifying 😂 

https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/portal/render_portlet.jsp#L455-L505

It is what it is 😂

**Comment by jbalsas** [comment]
Looks harmless enough... do you want us to forward it, @dgarciasarai? Not sure what's your team doing about merges at this moment.

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp:48]: OMG 😂

**Comment by dgarciasarai** [comment]
Hey @julien, @jbalsas, thanks for the review...And yes! In this case, we decided that this can be forwarded 😄

**Comment by julien** [comment]
ci:forward

**Comment by wincent** [comment]
Pull request #404?

not-found

**Pull request closed without merging**